### ISSUE TITLE
Imagenet dataset ?

### ISSUE THREAD

**Issue by luunguyen97** [mit-han-lab/amc-release#11]
Hi, thanks for your great job.
I wonder to know how can I get the right imagenet dataset you used for strategy search on mobilenet?

**Comment by tonylins** [comment]
We used the regular ImageNet dataset. You can find more information here: http://image-net.org/download

**Comment by luunguyen97** [comment]
@tonylins Did you mean the whole ILSVRC2012 dataset of ImageNet. I see that it very large dataset (about 150Gb for both training set and validation set). However, It is said in your AMC paper that you used 3000 images from the training set to evaluate the reward function. So I just need to download 3000 image of the dataset ?
Thanks.

**Comment by chaopengz** [comment]
> @tonylins Did you mean the whole ILSVRC2012 dataset of ImageNet. I see that it very large dataset (about 150Gb for both training set and validation set). However, It is said in your AMC paper that you used 3000 images from the training set to evaluate the reward function. So I just need to download 3000 image of the dataset ?
> Thanks.

Although the author used only 3000 images for reward evaluation, about 15000(300*50) images from training set is needed to do weights regression during pruning kernel. What's more, you need the whole training set to finetune the pruned model. Thus it is necessary to download the whole  ImageNet2012 dataset I think.
